### DISCUSSION TITLE
Sugestão: Indice dos Chafes de Fase e Monster Chef

### DISCUSSION THREAD

**Discussion by iurikill111** [YuriAlessandro/gerador-ficha-tormenta20#381] [Ideias]
Vejo que no Caverna do Saber só tem o Ozai dos Chefes de Fase, acho que seria legal adicionar os outros para ajduar mestres que estão procurando pois nem toda DB tras fichas de tormenta nessa matérias. O mesmo vale pros Monster Chefs.

**Comment by YuriAlessandro** [comment]
Sempre que meus olhos detectam, eu tento adicionar. Se for possível ajudar, um compiladinho por aqui dos que estão faltando ajuda. Nem sempre eu tenho muito tempo pra revisitar todas as revistas do passado.

PS: definitivamente eu não demorei muito para responder esse post.
